### DISCUSSION TITLE
DietPi-Software | Observium

### DISCUSSION THREAD

**Discussion by mews-se** [MichaIng/DietPi#6042] [Software Request]
#### Formal software information
- Software title | Observium
- Short description | Network monitoring via SNMP
- Official URL (if available) | https://www.observium.org/
- (Official) install documentation | https://docs.observium.org/

#### Are there similar/alternative software titles available with DietPi-Software?
- No

#### What makes your requested software better than the above solutions, if available?
- N/A

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?
- In all honesty, probably not that much since they provide an "install script" in bash.

#### Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? This is not needed, but could speed up our decision to implement it, as man power is always a topic :wink:.
- Yes, I could help out with this, e.g. if I experience issues.

**Comment by MichaIng** [comment]
Many thanks for your request. I converted in into a discussion. Everyone if you want to have this implemented vote for it via up bottom at above post.

**Comment by lutfor-diu** [comment]
nice software
